.github/workflows/integ-tests.yml

```diff
@@ -0,0 +1,211 @@
+          aws-region: ${{ env.AWS_DEFAULT_REGION }}
+      - name: Stop all left-over training jobs
+        run: |
+          aws sagemaker list-training-jobs --status-equals InProgress > running_jobs.json
+          jq -c '.[][]["TrainingJobName"]' running_jobs.json | while read i; do
+            jobName=`echo $i | cut -d "\"" -f 2`
+            echo "stopping training job $jobName"
+            aws sagemaker stop-training-job --training-job-name $jobName
+            sleep 5
+          done
+
+  launch_tensorboard_example:
+    uses: ./.github/workflows/run-syne-tune.yml
+    with:
+      script-path: examples/launch_tensorboard_example.py
+    secrets: inherit
+  
+  launch_tuning_gluonts:
+    needs: stop_all_training_jobs
+    uses: ./.github/workflows/run-syne-tune.yml
+    with:
+      script-path: examples/launch_tuning_gluonts.py
+      requires-aws-credentials: true
+    secrets: inherit
+
+  launch_plot_results:
+    uses: ./.github/workflows/run-syne-tune.yml
+    with:
+      additional-command: pip install matplotlib
+      script-path: examples/launch_plot_results.py
+    secrets: inherit
+
+  launch_rl_tuning:
+    uses: ./.github/workflows/run-syne-tune.yml
+    with:
+      additional-command: pip install -r examples/training_scripts/rl_cartpole/requirements.txt
+      script-path: examples/launch_rl_tuning.py
+    secrets: inherit
+
+  launch_simulated_benchmark:
+    uses: ./.github/workflows/run-syne-tune.yml
+    with:
+      script-path: examples/launch_simulated_benchmark.py
+    secrets: inherit
+
+  launch_standalone_bayesian_optimization:
+    uses: ./.github/workflows/run-syne-tune.yml
+    with:
+      script-path: examples/launch_standalone_bayesian_optimization.py
+    secrets: inherit
+
+  launch_moasha_instance_tuning:
+    needs: stop_all_training_jobs
+    uses: ./.github/workflows/run-syne-tune.yml
+    with:
+      script-path: examples/launch_moasha_instance_tuning.py
+      requires-aws-credentials: true
+    secrets: inherit
+
+  launch_nas201_transfer_learning:
+    uses: ./.github/workflows/run-syne-tune.yml
+    with:
+      additional-command: pip install matplotlib
+      script-path: examples/launch_nas201_transfer_learning.py
+    secrets: inherit
+
+  launch_nasbench201_simulated:
+    uses: ./.github/workflows/run-syne-tune.yml
+    with:
+      script-path: examples/launch_nasbench201_simulated.py
+    secrets: inherit
+
+  launch_pbt:
+    uses: ./.github/workflows/run-syne-tune.yml
+    with:
+      script-path: examples/launch_pbt.py
+    secrets: inherit
+
+  launch_height_sagemaker:
+    needs: stop_all_training_jobs
+    uses: ./.github/workflows/run-syne-tune.yml
+    with:
+      script-path: examples/launch_height_sagemaker.py
+      requires-aws-credentials: true
+    secrets: inherit
+
+  launch_height_standalone_scheduler:
+    uses: ./.github/workflows/run-syne-tune.yml
+    with:
+      script-path: examples/launch_height_standalone_scheduler.py
```

Comment:
Isnt this one fast?

Ok I see that some of the test appear in both this action and the "fast" action, what is the rationale for running the test twice?

Reply:
Good point, see comment https://github.com/awslabs/syne-tune/pull/520#discussion_r1095533413 for proposed fix